pkg/genericapiserver/genericapiserver.go

```diff
@@ -128,6 +131,20 @@ type APIGroupVersionOverride struct {
 	ResourceOverrides map[string]bool
 }
 
+// Info about an API group.
+type APIGroupInfo struct {
+	GroupMeta latest.GroupMeta
```

Comment:
This is a little different from `GroupVersionKind`, because TypeMeta.APIVersion would be of type GroupVersionKind, so we would need to keep a declaration of GroupVersionKind in the unversioned package. `GroupMeta` is not part of the API, so it doesn't belong to the unversioned package.

How about do a `type GroupMeta latest.GroupMeta` in the `genericapiserver` package?